addon/mocks/create.js

```diff
@@ -18,6 +18,10 @@ export const composeCreateMocksForSchema =
 
       let mocks = createMocks(typesAndMockFns, db, options);
 
+      if(options.scalars) {
+        mocks = Object.assign({}, mocks, options.scalars)
+      }
```

Comment:
2 quick notes here:
1. Please check for options. They are allowed to be null.
2. Please format the code slightly so it matches the rest of the codebase.

```javascript
if (options && options.scalars) {
  mocks = Object.assign({}, mocks, options.scalars);
}
```